pkg/thanos/statefulset.go

```diff
@@ -527,3 +545,61 @@ func mountSecret(secretSelector *v1.SecretKeySelector, volumeName string, trVolu
 	})
 	return mountpath
 }
+
+func generateRemoteWriteConfigYaml(remoteWrites []monitoringv1.RemoteWriteSpecV2) yaml.MapSlice {
```

Comment:
This function still needs some work as we would have to support all the fields from `RemoteWriteSpecV2` similar to how it's done in `generateRemoteWriteConfig`

Reply:
Refer to https://thanos.io/tip/components/rule.md/#stateless-ruler-via-remote-write, the user manual does not fully introduce all the fields, is it okay to complete the basic functions first?